azuredeploy.json

```diff
@@ -262,10 +262,12 @@
             "tenantId": "[subscription().tenantId]",
             "permissions": {
```

Comment:
Didn't know whether you wanted get for just the secret or not. Did both since it's a lab 👍